language/move-analyzer/editors/code/package-lock.json

```diff
@@ -1,7063 +1,7266 @@
 {
-    "name": "move-analyzer",
```

Comment:
It does not look like this whole file changed and yet this file's diff remove and then adds ALL lines. Is it due to formatting (e.g., file endings?). More generally, is it possible to have a more incremental diff here?

Reply:
> It does not look like this whole file changed and yet this file's diff remove and then adds ALL lines. Is it due to formatting (e.g., file endings?). More generally, is it possible to have a more incremental diff here?

Like windows newline problem, I will convert to Unix instead of